### ISSUE TITLE
Detail on Home Page 

### ISSUE THREAD

**Issue by Shalu411** [sanskrit-lexicon/COLOGNE#304]
Namaste
This dictionary is helpful - but lot of matter is inside undiscoverable. I wished, this kind of useful matter exited for click on home page upper pane- 
https://www.sanskrit-lexicon.uni-koeln.de/scans/csldev/csldoc/build/displays/index.html
The transliteration etc. as well- needs to be on every display - each dictionary- each page of search- so that people can easily access and use. Please rethink on remodeling a bit which has been long pending.

**Comment by gasyoun** [comment]
@Shalu411 I agree things are hard to grasp. But https://www.sanskrit-lexicon.uni-koeln.de/scans/csldev/csldoc/build/displays/index.html itself needs to be documented. You have tried to document what issues you have come across as a new comer, but got tired of it. I guess @funderburkjim does not minds at all, but it's up to as to decide how to document and where exactly to hyperlink.

**Comment by funderburkjim** [comment]
In order for me (or someone else) to incorporate these UI suggestions,   the suggestions need to be more specific.   Currently, based on your comment above, I have only a vague idea of what you are envisioning.  

Maybe you could make a 'mock-up'  that would give exactly what you think the user should see, and what a given click should do.   Such a mock-up (or design proposal) would help transfer the thoughts in your mind into my mind.   

If you want to proceed further, I suggest you start with whatever you think would be the simplest improvement.   Describe it in elaborate detail, with pictures, and flow-charts and textual explanations -- Make the idea as real as you can.  As fully as you can, describe what would need to be done to accomplish your idea.

If your level of interest is very high,  then you could even begin to learn some of the tools of web site design - such as HTML and CSS.

**Comment by drdhaval2785** [comment]
@Shalu411 Any update? What do you want? You can draw how you want to get it displayed.
